EasyPost.Tests/HttpTests/ParametersTest.cs

```diff
@@ -0,0 +1,88 @@
+using System.Collections.Generic;
+using EasyPost.Http;
+using EasyPost.Tests._Utilities.Annotations;
+using Xunit;
+
+namespace EasyPost.Tests.HttpTests
+{
+    public class ParametersTest
+    {
+        #region Tests
+
+        [Fact]
+        [Testing.Function]
+        public void TestWrapParametersDictionary()
+        {
+            Dictionary<string, object> dictionaryToWrap = new()
+            {
+                { "foo", "bar" },
+                { "baz", "qux" }
+            };
+
+            Dictionary<string, object> wrappedDictionary = dictionaryToWrap.Wrap("top", "middle", "bottom");
+
+            Dictionary<string, object> expectedDictionary = new()
```

Comment:
Question: The formatting on this is awful, can we not condense it any?

Reply:
This is as condensed as it can be, per our previous attempts at styling dictionaries.